### PR TITLE
tests: Fix args and test viscoacoustic

### PR DESCRIPTION
**Pull request by georgebisbas**
Viscoacoustic examples is currently failing by default as its kernels are not in the usual ones. Adding explicit tests for them and restoring its working status.

### CODE REVIEW

File: examples/seismic/utils.py [hunk 1]
```diff
@@ -192,7 +192,8 @@ def seismic_args(description):
     parser.add_argument("--nbl", default=40,
                         type=int, help="Number of boundary layers around the domain")
     parser.add_argument("-k", dest="kernel", default='OT2',
-                        choices=['OT2', 'OT4', 'centered', 'staggered'],
+                        choices=['OT2', 'OT4', 'centered', 'staggered',
+                                 'blanch_symes', 'ren', 'deng_mcmechan'],
```
Comment (FabioLuporini) [inline examples/seismic/utils.py:196]: these ones only apply to viscoacoustic

File: examples/seismic/viscoacoustic/wavesolver.py [hunk 2]
```diff
@@ -32,6 +32,9 @@ def __init__(self, model, geometry, space_order=4, kernel='blanch_symes', **kwar
 
         self.space_order = space_order
         self.dt = self.model.critical_dt
+        if kernel not in {'blanch_symes', 'ren', 'deng_mcmechan'}:
```
Comment (FabioLuporini) [inline examples/seismic/viscoacoustic/wavesolver.py:35]: no...this should be made clean by having viscoacoustic use its own arg for kernel

File: examples/seismic/utils.py (hunk 1 above)
Comment (georgebisbas) [reply examples/seismic/utils.py:196]: So now, after migrating options to seismic_args we need to find a more 'global' way to do that, how should that be handled?

File: examples/seismic/viscoacoustic/wavesolver.py (hunk 2 above)
Comment (georgebisbas) [reply examples/seismic/viscoacoustic/wavesolver.py:35]: Out of seismic_args I guess?

File: examples/seismic/utils.py (hunk 1 above)
Comment (mloubout) [reply examples/seismic/utils.py:196]: Yes, same way `--fs` or `--noazimut` is done in acoustic and tti, example specific args in each example with according checks.

File: examples/seismic/viscoacoustic/viscoacoustic_example.py [hunk 3]
```diff
@@ -41,6 +43,14 @@ def test_viscoacoustic():
     assert np.isclose(norm(rec), 18.7749, atol=1e-3, rtol=0)
 
 
+@pytest.mark.parametrize('ndim', [1, 2, 3])
+def test_viscoacoustic_stability(ndim):
```
Comment (mloubout) [inline examples/seismic/viscoacoustic/viscoacoustic_example.py:47]: add kernel as parameter to test all three

File: examples/seismic/utils.py (hunk 1 above)
Comment (georgebisbas) [reply examples/seismic/utils.py:196]: But kernel options needs to be in a dict right? It is not an extra arg.

File: examples/seismic/viscoacoustic/viscoacoustic_example.py (hunk 3 above)
Comment (FabioLuporini) [reply examples/seismic/viscoacoustic/viscoacoustic_example.py:47]: is this the only thing missing here? @mloubout @georgebisbas

Comment (mloubout) [reply examples/seismic/viscoacoustic/viscoacoustic_example.py:47]: I fixed the other ones so if wanted yes, can always be added later if wanted and just merge

Comment (FabioLuporini) [reply examples/seismic/viscoacoustic/viscoacoustic_example.py:47]: still missing?

Comment (georgebisbas) [reply examples/seismic/viscoacoustic/viscoacoustic_example.py:47]: Yes it is still missing.

**Comment by FabioLuporini** [comment]
Merged.

**Pull request merged**